examples/WORKSPACE

```diff
@@ -12,7 +12,7 @@ local_repository(
 # git_repository(
 #     name = "bazelruby_ruby_rules",
 #     remote = "https://github.com/yugui/rules_ruby.git",
-#     tag = "v0.1.0",
+#     tag = "v0.1.0",`
```

Comment:
To test different runtimes and rules each example should have a different workspace. so this WORKSPACE file should not exist here.